packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -839,6 +856,7 @@ export function TransferPanel() {
             l1Signer,
             txLifecycle: {
               onTxSubmit: () => {
+                setTransactionHistoryTab(TransactionHistoryTab.DEPOSITS)
```

Comment:
NIT: To simplify things, would it be possible to optionally pass the tab-name (Defaulting to `DEPOSITS`) in `openTransactionHistoryPanel()` function, and then it opens with the mentioned tab-name,

eg. `openTransactionHistoryPanel({tabName: TransactionHistoryTab.DEPOSITS, isCctpTab: false})`


Instead of calling `setTransactionHistoryTab`, `openTransactionHistoryPanel`, `showCctpDepositsTransactions` functions separately?

Reply:
Tbh tx history tabs will be merged together and all this will be removed in the incoming PRs so not sure if we need to worry about maintainability here

Reply:
Although that's just regarding cctp, we could still have something for deposits/withdrawals

Reply:
For CCTP, we have `showCctpDepositsTransactions` already